docs/release/release_notes_v2_2.md

```diff
@@ -0,0 +1,128 @@
+# DAOS Version 2.2 Release Notes
+
+**DRAFT** **DRAFT** **DRAFT** **DRAFT** **DRAFT**
+
+We are pleased to announce the release of DAOS version 2.2.
+
+
+## DAOS Version 2.2.0 (2022-09-xx)
+
+### General Support
+
+This release adds the following changes to the DAOS support matrix:
+
+- Rocky Linux 8 support is added.
+- CentOS Linux 8 support is removed.
+- Support for the `libfabric/tcp` provider is added (replaces `libfabric/sockets`)
+- UCX support is added (Technology Preview - not recommended in production environments)
+
+For a complete list of supported hardware and software, refer to the
+[Support Matrix](https://docs.daos.io/v2.2/release/support_matrix/).
+
+### Key features and improvements
+
+#### EL8 Support Transition
+
+CentOS Linux 8 support has been removed, and has been replaced by Rocky Linux 8 support.
+RHEL 8.5 is no longer supported by RedHat; RHEL 8.6 and RHEL 8.4 EUS are still supported.
+
+#### New Network Providers
+
+- Support for the `libfabric/tcp` provider is added.
+  This replaces the `libfabric/sockets` provider that was supported with DAOS 2.0.
+  The `tcp` provider achieves significantly higher performance than the `sockets` provider
+  and is the recommended provider for non-RDMA networks.
+
+- UCX support is added as a Technology Preview.
+  Refer to [UCX](https://docs.daos.io/v2.2/admin/ucx/) for details.
+  It is a roadmap item to fully support UCX in the DAOS 2.4 release.
+
+#### Usability Improvements
+
+This release adds the following usability improvements:
+
+- Interoperability of DAOS 2.2 with DAOS 2.0.
+
+- The required number of hugepages on DAOS servers is now auto-calculated.
+  It is no longer necessary (and discouraged) to specify `nr_hugepages` in the
+  `daos_server.yml` configuration file.
+
+- Intel VMD devices are now supported in the control plane.
+  Roadmap items exist for future DAOS releases to add
+  support for LED management of VMD-managed NVMe disks,
+  and hot-plug support for VMD-managed NVMe disks.
+
+- The `dmg pool create` command now accepts pool size specification in percent.
```

Comment:
suggest that we state that it's "in addition to using the arbitrary numbers"

Reply:
done